pkg/aws/awscluster/mutate_awscluster.go

```diff
@@ -204,6 +219,43 @@ func (m *Mutator) MutatePodCIDR(awsCluster infrastructurev1alpha2.AWSCluster) ([
 	return result, nil
 }
 
+// MutateMasterPreHA is there to mutate the master instance attributes of the AWSCluster CR in legacy versions.
+// This can be deprecated once no versions < 11.4.0 are in use anymore
```

Comment:
Let's have a TODO issue for that so we can track it in our board and link it here in the code for cross reference.

Reply:
I'll take care of it :)